examples/example-xds/README.md

```diff
@@ -10,19 +10,43 @@ will be very hard to produce a working enviornment just by this example. Please
 refer to documentation specific for your XDS management server and
 environment.__
 
-The example requires grpc-java to already be built. You are strongly encouraged
-to check out a git release tag, since there will already be a build of grpc
-available. Otherwise you must follow [COMPILING](../../COMPILING.md).
+The example requires grpc-xds, but grpc-xds is not currently being published.
+You will thus need to build it yourself. This should guide you, but if you
+encounter issues please consult [COMPILING.md](../../COMPILING.md).
 
 ### Build the example
 
-1. Build the hello-world example server or the hostname example server. See
+1. The server does not use XDS, so recent releases work fine. Building using
+recent releases is much easier, so check out the most recent release tag:
+```
+$ git checkout v1.27.0
```

Comment:
This number and below will be changed after every release. Add an line of instruction at https://github.com/grpc/grpc-java/pull/6631/files#diff-60cd2f42437b5cb3c2c09391a6f4c54aR125 ?

Reply:
If you look at the history of https://github.com/grpc/grpc-java/commits/master/documentation/android-channel-builder.md, we miss a commit "Update README etc to reference 1.27.0", but RELEASING.md doesn't document to add this commit in the master branch, while similar commits were added for the past releases. I'm confused.

Reply:
As far as I'm concerned, I'm okay if we don't update this. Two reasons: 1) it'll work fine, no big deal. 2) grpc-xds should be published the next release or the release after, so it will only be "wrong" for a maximum of one release. Once grpc-xds is published this will be rewritten to look like the other examples and just say "use the latest tag"